COLLABORATOR_GUIDE.md

```diff
@@ -9,3 +9,6 @@ with the following differences:
   - There's no minimum wait time for landing a pull request. Pull requests can
     land as soon as they get approved by at least one Collaborator.
   - Approval must be from Collaborators who are not authors of the change.
+  - If the pull request was open for three days (72 hours) without reviews, and
```

Comment:
Not to intentionally slow llnode down any further, but what about 7 days to match the one approval clause from https://github.com/nodejs/node/blob/master/COLLABORATOR_GUIDE.md#code-reviews.

Reply:
I'm not entirely opposed to 7 days, but I don't think "to match the one approval clause from core" is a good reason to do so. I'm open to hearing other arguments to use 7 days instead of 3 though.

On a counter-argument to 7 days, if there is a bug and a PR with a fix doesn't get a review, it will now take seven days to release a fix. We could have a shorter waiting period for bug fixes, but that seems to overly complicate the policy.

Reply:
To counter your counter-argument, if there is a bug under this proposal, it will take 3 days to release a fix 😄. I guess any amount of waiting will be an arbitrary amount. I only recommended going with 7 days because that was the agreed upon answer to this exact same problem in core. At the end of the day, you're the one doing literally all of the work, and I'm not going to block this PR.